.expeditor/build-docker-images.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+export DOCKER_CLI_EXPERIMENTAL=enabled
+
+arch=$1
+
+echo "--- Building chef/chef:${EXPEDITOR_VERSION} docker image for ${arch}"
+docker build \
+  --build-arg "CHANNEL=${EXPEDITOR_CHANNEL}" \
+  --build-arg "VERSION=${EXPEDITOR_VERSION}" \
+  -t "chef/chef:${EXPEDITOR_VERSION}-${arch}" .
+
+echo "--- Pushing chef/chef:${EXPEDITOR_VERSION} docker image for ${arch} to dockerhub"
+docker push "chef/chef:${EXPEDITOR_VERSION}-${arch}"
+
+echo "--- Creating chef/chef:${EXPEDITOR_VERSION} multiarch manifest"
+docker manifest create "chef/chef:${EXPEDITOR_VERSION}" \
```

Comment:
This `docker manifest create` command is only amending one architecture image in each build step and then that manifest is getting pushed in each build step. Does the final manifest somehow end up including both architecture images?

Reply:
Yes, this is exactly how it works - think of manifests as just a new tag and every amend just adds to that tag and updates it in place in dockerhub